woocommerce/navigation
# Hide admin bar on wc nav pages

Fixes #100 

Removes the admin bar in the new nav experience.  Note that this is hidden via CSS since the `show_admin_bar` filter returns early with `true` when on any admin page.

Follow-up for WCA header and breadcrumbs to come in the WCA repo.

### Screenshot
<img width="618" alt="Screen Shot 2020-09-22 at 2 42 24 PM" src="https://user-images.githubusercontent.com/10561050/93924120-fb955180-fd1c-11ea-95e0-7d3275e48a82.png">


### Testing

1. Visit various pages in the new nav experience.
1. Make sure the layout is correct and the admin bar is hidden.
1. Visit non-wc pages and make sure things look okay.
